src/Plugins/SimplnxCore/src/SimplnxCore/Filters/ExtractPipelineToFileFilter.cpp

```diff
@@ -0,0 +1,151 @@
+#include "ExtractPipelineToFileFilter.hpp"
+
+#include "simplnx/Common/AtomicFile.hpp"
+#include "simplnx/Common/StringLiteral.hpp"
+#include "simplnx/Parameters/FileSystemPathParameter.hpp"
+#include "simplnx/Parameters/StringParameter.hpp"
+#include "simplnx/Pipeline/Pipeline.hpp"
+#include "simplnx/Utilities/Parsing/DREAM3D/Dream3dIO.hpp"
+
+#include "simplnx/Utilities/SIMPLConversion.hpp"
+
+#include <nlohmann/json.hpp>
+
+namespace
+{
+constexpr nx::core::StringLiteral k_ImportedPipeline = "Imported Pipeline";
+} // namespace
+
+namespace nx::core
+{
+//------------------------------------------------------------------------------
+std::string ExtractPipelineToFileFilter::name() const
+{
+  return FilterTraits<ExtractPipelineToFileFilter>::name;
+}
+
+//------------------------------------------------------------------------------
+std::string ExtractPipelineToFileFilter::className() const
+{
+  return FilterTraits<ExtractPipelineToFileFilter>::className;
+}
+
+//------------------------------------------------------------------------------
+Uuid ExtractPipelineToFileFilter::uuid() const
+{
+  return FilterTraits<ExtractPipelineToFileFilter>::uuid;
+}
+
+//------------------------------------------------------------------------------
+std::string ExtractPipelineToFileFilter::humanName() const
+{
+  return "Extract DREAM.3D Pipeline To File";
+}
+
+//------------------------------------------------------------------------------
+std::vector<std::string> ExtractPipelineToFileFilter::defaultTags() const
+{
+  return {className(), "IO", "Input", "Read", "Import", "Output", "Write", "Export"};
+}
+
+//------------------------------------------------------------------------------
+Parameters ExtractPipelineToFileFilter::parameters() const
+{
+  Parameters params;
+  params.insertSeparator(Parameters::Separator{"Input Parameters"});
+  params.insert(std::make_unique<FileSystemPathParameter>(k_ImportFileData, "Import File Path", "The DREAM.3D file path the pipeline should be taken from", FileSystemPathParameter::ValueType{},
```

Comment:
params.insert(std::make_unique<FileSystemPathParameter>(k_ImportFileData, "Input DREAM3D File Path", "The file path to the .dream3d that holds the pipeline to be extracted.", FileSystemPathParameter::ValueType{},